src/VisualStudio/CSharp/Test/CodeModel/AbstractFileCodeElementTests.cs

```diff
@@ -43,11 +43,6 @@ protected TestWorkspace GetWorkspace()
             return WorkspaceAndCodeModel.workspace;
         }
 
-        private VisualStudioWorkspace GetExtraWorkspaceToDisposeButNotUse()
```

Comment:
The VisualStudioWorkspace instance is now created by the MEF container, which will dispose it when we dispose the container itself.